### PR TITLE
Rails 5 Support

### PR DESCRIPTION
**Pull request by iloveitaly**
Loosing the specifications for minitest-rails seems to be all that's needed to support rails5!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 doc
 pkg
 log
+Gemfile.lock
```
Comment (blowmage) [inline .gitignore]: Please remove these two lines.

**Comment by mbianco-stripe** [comment]
@blowmage just updated! Also loosened capybara dependency to work with capybara 3.

File: minitest-rails-capybara.gemspec [hunk 2]
```diff
@@ -23,22 +23,22 @@ Gem::Specification.new do |s|
     s.specification_version = 4
 
     if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
-      s.add_runtime_dependency(%q<minitest-rails>.freeze, ["~> 3.0"])
-      s.add_runtime_dependency(%q<capybara>.freeze, ["~> 2.7"])
+      s.add_runtime_dependency(%q<minitest-rails>.freeze, [">= 3.0", "<= 5.2"])
```
Comment (blowmage) [inline minitest-rails-capybara.gemspec]: This should remain `~> 3.0`.

Comment (iloveitaly) [reply minitest-rails-capybara.gemspec]: Why is that? This gem continues to work with more recent `minitest-rails` versons.

Comment (blowmage) [reply minitest-rails-capybara.gemspec]: minitest-rails-capybara 3.x is intended to work with minitest-rails 3.x. (Both minitest-rails 3.x and minitest-rails 5.0.x support Rails 5.x). If you want a version for minitest-rails 5.0.x then we should release minitest-rails-capybara 5.0.x, but I am hesitant to do that.

I discuss my thoughts about versioning in this [blog post](https://blowmage.com/2019/06/18/minitest-rails-releases).

Comment (blowmage) [reply minitest-rails-capybara.gemspec]: Ping.

Comment (iloveitaly) [reply minitest-rails-capybara.gemspec]: @blowmage updated!

**Pull request merged**